app/models/charge.js

```diff
@@ -89,26 +89,22 @@ module.exports = correlationId => {
     return response.body
   }
 
-  const patch = function (chargeId, op, path, value, subSegment) {
-    return new Promise(function (resolve, reject) {
+  const patch = async function (chargeId, op, path, value, subSegment) {
+    try {
       const payload = {
         op: op,
         path: path,
         value: value
       }
-      connectorClient({ correlationId }).patch({ chargeId, payload }, subSegment)
-        .then(response => {
-          const code = response.statusCode
-          if (code === 200) {
-            resolve()
-          } else {
-            reject(new Error('Calling connector to patch a charge returned an unexpected status code'))
-          }
-        })
-        .catch(err => {
-          reject(err)
-        })
-    })
+      const response = await connectorClient({ correlationId }).patch({ chargeId, payload }, subSegment)
+      if (response.statusCode === 200) {
+        return
+      } else {
+        throw new Error('Calling connector to patch a charge returned an unexpected status code')
```

Comment:
could inverse the if statement so we just throw an error if the status code isn't 200, the we can do without the return statement

Reply:
fixed

Reply:
@sfount As the old code did a `reject(new Error('Calling connector to patch a charge returned an unexpected status code'))` I don't think anything's been changed here.